bridge_node:
  ros__parameters:
    poll_hz: 25
    server_ip: "192.168.0.200"
    timeout: 2.0